index.js

```diff
@@ -0,0 +1,219 @@
+const cards = [
```

Comment:
Please use UPPER_CASE for all constant literals
```suggestion
const CARDS = [
```